scripts/package_examples.rb

```diff
@@ -53,11 +50,13 @@ def replace_in_file(filepath, pattern, replacement)
 end
 
 framework_directory_for_example = {
-  'examples/ios/xcode-7/objc' => '../../../../ios/static/xcode-7',
+  'examples/ios/objc' => '../../../ios/static',
   'examples/osx/objc' => '../../../osx',
   'examples/tvos/objc' => '../../../tvos',
   'examples/ios/swift-2.2' => '../../../ios/swift-2.2',
-  'examples/tvos/swift-2.2' => '../../../tvos',
+  'examples/tvos/swift-2.2' => '../../../tvos/swift-2.2',
```

Comment:
There's non-packaging work needed for them to build, and I don't think it's worth fixing that now. Essentially it's unclear how to tell `Realm.xcodeproj` to use Swift 2.3 from the Xcode GUI, without completely changing how it's built (i.e. building it via `build.sh` rather than Xcode like it is now).